pkgs/development/perl-modules/generic/builder.sh

```diff
@@ -17,7 +19,14 @@ preConfigure() {
             first=$(dd if="$fn" count=2 bs=1 2> /dev/null)
             if test "$first" = "#!"; then
                 echo "patching $fn..."
-                sed -i "$fn" -e "s|^#\!\(.*[ /]perl.*\)$|#\!\1$perlFlags|"
+                # TODO, handle this better?
+                # `perl5.28.1-Test-Simple-1.302141` bakes line numbers into the expected errors in tests
+                # inserting a "use lib ...;\n" into the file offsets everything and makes the tests fail
+                if [[ "$name" == *"Test-Simpl"* ]]; then
```

Comment:
we're going to have to fix TestSimple before merging this :)

Reply:
Perhaps we can just disable it's tests until we have a better solution.

Reply:
better solution, just patch things after testing!